e2e-tests/vars.sh

```diff
@@ -20,6 +20,8 @@ export BUCKET=${BUCKET:-"pg-operator-testing"}
 export PMM_SERVER_VERSION=${PMM_SERVER_VERSION:-"9.9.9"}
 export IMAGE_PMM_CLIENT=${IMAGE_PMM_CLIENT:-"perconalab/pmm-client:dev-latest"}
 export IMAGE_PMM_SERVER=${IMAGE_PMM_SERVER:-"perconalab/pmm-server:dev-latest"}
+export PGOV1_TAG="1.4.0"
```

Comment:
Do we need this fixed here? Can we have it dynamic and not in general vars but somewhere in the test code?

Reply:
It is being used in several steps of the test case, i.e. in several .yaml files. I would have to initialize it in each and every one of them, because kuttl starts a new shell for every yaml file. This is the latest version tag for the latest 1.x version and is basically isn't going to change anytime, because we have already released PG operator version 1.x.

Reply:
you can catch it in the first yaml file, store in `${TEMP_DIR}/pgov1_tag.txt` file and then just read it where you need it.